app-portage/repoman/README.md

```diff
@@ -0,0 +1,3 @@
+This is a fork of app-portage/repoman package. The sole reason for
+having it here is to drop support for python 3.8 and 3.9 we haven't
+yet packaged.
```

Comment:
Just a note for me.
In the future, we should really update eclass files to make them accept python 3.8 and 3.9.
It has been a huge pain for everyone who touches the code.

Reply:
That may mean custom modification of the eclass, because the updated upstream version may, for example, drop support for python2 we still package or for old EAPI we still use.